src/styles/app.scss

```diff
@@ -324,3 +324,55 @@ header h1 {
   }
 }
 
+@media (prefers-color-scheme: dark) {
+  $yellow: #835f00;
+
+  body {
+    background-color: #181a1b;
```

Comment:
For maintainability reasons, wouldn't it be better to instead define variables and override them in the `@media (prefers-color-scheme: dark)` and have a common CSS ode in any case using these variables? (that's what we do in rustdoc for example)

Reply:
another alternative would be using https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark and potentially have it polyfilled for browsers that don't support it natively yet (that's what we do in crates.io 😉)

Reply:
The support is way too recent. Why bother with yet another tool (polyfill) when it can just work? Also, what happens if you want to support more than 2 themes? Anyway, seems interesting but too recent and too limited to be worth it.

Reply:
Using CSS variables is a good suggestion. After thinking about it for a sec, I'd rather do that in a follow-up PR because it's a more invasive change:
- find common colors in both light/dark mode
- find suitable names for them
- refactor the stylesheet

I'm afraid all these changes would block this patch on a review for too long.

Reply:
You can start by adding variables for just the cases you want here and we can eventually check for a follow-up later on. What do you think?

Reply:
ok. tried assigning some names. not completely happy but hope it's ok for now